package.json

```diff
@@ -9,6 +9,7 @@
 		"build-release": "tsc -p src",
 		"prepublish": "safe-publish-latest && npm run build-release",
 		"test": "mocha",
+		"test-published-files": "rm -rf \"ecmarkup-$npm_package_version.tgz\" package && npm pack && tar zxvf \"ecmarkup-$npm_package_version.tgz\" && cp -r test package/test && cd package && ../node_modules/.bin/mocha && cd .. && rm -rf \"ecmarkup-$npm_package_version.tgz\" package",
```

Comment:
Did you try invoking `mocha` directly? Why wouldn't that work?

Reply:
Oh, I just always forget that's a thing. Updated.